docs/xroad_security_server_toolkit_user_guide.md

```diff
@@ -1500,6 +1506,60 @@ Listing all the diagnostics can be done with ```xrdsst diagnostics all```
 
 This command will list all the information from 4.2.12.1 - 4.2.12.3
 
+#### 4.2.13 Keys management
+
+##### 4.2.13.1 List keys
+
+* Access rights: XROAD_SECURITY_OFFICER
+
+Listing certificate keys can be done with:
+```
+xrdsst key list --ss <SECURITY_SERVER_NAME> --token <TOKEN_ID>
+```
+
+* <SECURITY_SERVER_NAME> seccurity server name, e.g. ss1
+* <TOKEN_ID> token id, multiple values can also be given, separated by comma, e.g. 0,1
+
+╒══════════════════════════════════════════╤════════════════════════════════╤════════════════════════════════╤════════════════╤═══════════════════════════════════════════════╤════════════╕
+│ ID                                       │ LABEL                          │ NAME                           │ USAGE          │ POSSIBLE ACTIONS                              │   Nº CERTS │
+╞══════════════════════════════════════════╪════════════════════════════════╪════════════════════════════════╪════════════════╪═══════════════════════════════════════════════╪════════════╡
+│ 61F82DF2B7E1A43DF500FC3E7C8AE4B6D2DD0C7E │ ss1-default-auth-key           │ ss1-default-auth-key           │ AUTHENTICATION │ DELETE, EDIT_FRIENDLY_NAME, GENERATE_AUTH_CSR │          1 │
+├──────────────────────────────────────────┼────────────────────────────────┼────────────────────────────────┼────────────────┼───────────────────────────────────────────────┼────────────┤
+│ D6EFFF21413B0A6974D087949995B4C677DFD8D1 │ ss1-default-sign-key           │ ss1-default-sign-key           │ SIGNING        │ DELETE, EDIT_FRIENDLY_NAME, GENERATE_SIGN_CSR │          1 │
+├──────────────────────────────────────────┼────────────────────────────────┼────────────────────────────────┼────────────────┼───────────────────────────────────────────────┼────────────┤
+│ 7D98B5BCF30F59351D9D396EF350AE3899286927 │ ss1-default-sign-key_COM_12345 │ ss1-default-sign-key_COM_12345 │ SIGNING        │ DELETE, EDIT_FRIENDLY_NAME, GENERATE_SIGN_CSR │          1 │
+╘══════════════════════════════════════════╧════════════════════════════════╧════════════════════════════════╧════════════════╧═══════════════════════════════════════════════╧════════════╛
+
+* ID Id of the key
+* LABEL label of the key
+* NAME friendly name of the key
+* USAGE type of certificate that can be used with the key
+* POSSIBLE ACTIONS List of possible actions that can be done to the key sepparated by comma
+* Nº CERTS number of certificates added to the key
+
+##### 4.2.13.2 Update keys
+
+* Access rights: XROAD_SECURITY_OFFICER
+
+The friendly name of a key can be updated with:
+```
+xrdsst key update --ss <SECURITY_SERVER_NAME> --token <TOKEN_ID> --name <FRIENDLY_NAME>
```

Comment:
Required parameters are ss, key and name, not token

Reply:
This is not resolved...you still have "--token" as parameter, but the operation requires "--key" as parameter